torch/ao/quantization/fake_quantize.py

```diff
@@ -352,8 +352,12 @@ def forward(self, X: torch.Tensor) -> torch.Tensor:
 Default dynamic fake_quant for activations.
 """
 
-default_symmetric_fixed_qparams_fake_quant = FixedQParamsFakeQuantize.with_args(observer=default_symmetric_fixed_qparams_observer)
-default_affine_fixed_qparams_fake_quant = FixedQParamsFakeQuantize.with_args(observer=default_affine_fixed_qparams_observer)
+default_symmetric_fixed_qparams_fake_quant = (
```

Comment:
would be good to change this name as well

Reply:
will do